mochitest.sh

```diff
@@ -14,9 +14,10 @@ cd /activity-stream && npm install . && npm run buildmc
 # Build latest m-c with Activity Stream changes
 cd /mozilla-central && ./mach build \
   && ./mach lint -l codespell browser/components/newtab \
+  && ./mach test browser/components/newtab/test/browser --headless \
+  && ./mach test browser/components/newtab/test/xpcshell \
```

Comment:
This looks like it would regress the `--log-tbpl test_run_log` check for actually failing. Unless `mach test` correctly fails now ?

Reply:
Actually, looks like failing test should cause the command to have non-zero exit code:

https://bugzilla.mozilla.org/show_bug.cgi?id=1473392